fiscalyear.py

```diff
@@ -226,10 +226,16 @@ def _check_quarter(quarter):
         )
 
 
-class FiscalYear(object):
+class _Hashable(object):
+    def __hash__(self):
+        return hash(tuple(map(lambda x: getattr(self, x), self.__slots__)))
+
+
+class FiscalYear(_Hashable):
     """A class representing a single fiscal year."""
 
-    __slots__ = "_fiscal_year"
+    __slots__ = "_fiscal_year",
```

Comment:
```suggestion
    __slots__ = ["_fiscal_year"]
```
This is probably more clear and uniform

Reply:
https://github.com/adamjstewart/fiscalyear/pull/24/commits/ba817e239c99dbb56e98ede806f2045eee2a9eec